### PR TITLE
Poker Hands Cheatsheet

### PR DESCRIPTION
**Pull request by jee1mr**
**What does your Instant Answer do?**
It provides CheatSheet for Poker Hands.

**What problem does your Instant Answer solve (Why is it better than organic links)?**
It's better because it's a Zero Click Info

**What is the data source for your Instant Answer? (Provide a link if possible)**
[Wikipedia](https://en.wikipedia.org/wiki/List_of_poker_hands)

**Why did you choose this data source?**
Reliable and constantly updated by the community.

**Are there any other alternative (better) data sources?**
Not required.

**What are some example queries that trigger this Instant Answer?**
"poker hands cheatsheet"
"poker hands shortcut"

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Gamers (online/offline)

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
It's taken from the [ideas list](https://duck.co/forum/thread/14080/quack-hack-bangalore) provided by @zekiel for a meetup event.

**Which existing Instant Answers will this one supercede/overlap with?**
None

**Are you having any problems? Do you need our help with anything?**
The appearance doesn't look neat to me, may be I need to do some changes. Also, I need help with representing it in a better way, if possible.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
poker

-----
IA Page: https://duck.co/ia/view/poker_hands_cheat_sheet

### CODE REVIEW

**Comment by jee1mr** [comment]
I'll fill the template soon.

**Comment by MariagraziaAlastra** [comment]
Thanks @jee1mr, nice work! We'll get back to you soon, just need to have the design team sign-off for this :)

**Comment by jee1mr** [comment]
@MariagraziaAlastra Cool. It's the first time I'm doing a cheatsheet. The appearance of my cheatsheet looks somewhat ugly to me.

**Comment by zekiel** [comment]
@jee1mr  i love this -- tho agree on the text. Would maybe just use the text descriptors and leave out the examples.

**Comment by jee1mr** [comment]
@zekiel Looks better?
pok1
pok2

**Comment by zekiel** [comment]
I agree. We're also working on additional options for the visuals so can be converted in the future.

File: share/goodie/cheat_sheets/poker-hands.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+    "id": "poker_hands_cheat_sheet", 
+    "name": "Poker",
+    "description": "Game",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/poker-hands.json]: Let's make this a little more descriptive? Or, we can remove it altogether. It's not required :)

File: share/goodie/cheat_sheets/poker-hands.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        "sourceName": "Wikipedia",
+        "sourceUrl" : "https://en.wikipedia.org/wiki/List_of_poker_hands"
+    },
+    "section_order": [  
+        "Poker Hands"
+    ],
+    "sections": {       
+        "Poker Hands": [
+            {
+                "key": "Straight flush",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/poker-hands.json]: Looks like Royal Flush is missing?

File: share/goodie/cheat_sheets/poker-hands.json (hunk 1 above)
Comment (jee1mr) [inline share/goodie/cheat_sheets/poker-hands.json]: I think we should mention somewhere that the list is in the order of priority. What do you think?

File: share/goodie/cheat_sheets/poker-hands.json (hunk 2 above)
Comment (jee1mr) [inline share/goodie/cheat_sheets/poker-hands.json]: @moollaza They call it Straight Flush. I think we could change it to Royal Flush. Don't know why it's so in Wikipedia. 
(https://en.wikipedia.org/wiki/Royal_flush_(poker_hand)) redirects to Straight Flush as well.

File: share/goodie/cheat_sheets/poker-hands.json (hunk 1 above)
Comment (moollaza) [inline share/goodie/cheat_sheets/poker-hands.json]: Yes! That'd be great for the description :)

File: share/goodie/cheat_sheets/poker-hands.json (hunk 2 above)
Comment (moollaza) [inline share/goodie/cheat_sheets/poker-hands.json]: I think we should mention Royal Flush separately. It’s a variant of the Straight Flush, but outranks all other Straights.

>An ace-high straight flush such as A♦ K♦ Q♦ J♦ 10♦ is known as a royal flush, and is the highest-ranking standard poker hand. It is usually treated as a distinct hand in video poker[citation needed]. There are four possible royal flushes: one of each suit.

Comment (jee1mr) [inline share/goodie/cheat_sheets/poker-hands.json]: Right, didn't realize that.

**Comment by jee1mr** [comment]
@moollaza 
pok3

File: share/goodie/cheat_sheets/poker-hands.json (hunk 1 above)
Comment (moollaza) [inline share/goodie/cheat_sheets/poker-hands.json]: @jee1mr looks like this still needs to be improved!

**Comment by moollaza** [comment]
@jee1mr just the description!

**Comment by moollaza** [comment]
@jee1mr whoops! I didn't know you updated this! Please be sure to let us know when you've made changes otherwise we won't get notifications.

This LGTM :+1:

**Comment by moollaza** [comment]
@jee1mr somehow I forgot to notify you about this one!

congrats! It's live: https://duckduckgo.com/?q=poker+hands+cheat+sheet&t=ffab&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**